frontend/src/components/organizations/OrganizationInfo.vue

```diff
@@ -221,50 +224,146 @@ export default {
       default: false,
     },
   },
+  emits: ['update:loading'],
   data() {
     return {
-      isFormComplete: false,
-      organizationReadOnly: undefined,
-      organizationEdit: undefined,
+      organizationEdit: {},
       localLoading: false,
-      rules,
       editMode: false,
+      uploadedDocuments: [],
+      uploadedDocumentsInitialValue: [],
+      documentsToUpload: [],
+      documentsToDelete: [],
+      showUploadDocumentsAlert: false,
     }
   },
-  computed: {
-    ...mapState(useAuthStore, ['userInfo']),
+  watch: {
+    documentsToUpload: {
+      handler(val) {
+        if (val.length === 0 && this.uploadedDocuments.length === 0 && !this.isLoading) {
+          this.showUploadDocumentsAlert = true
+        } else {
+          this.showUploadDocumentsAlert = false
+        }
+      },
+    },
+    'organizationEdit.hasInclusionPolicy': {
+      handler(val, oldVal) {
+        if (oldVal === true && val === false) {
+          this.documentsToDelete = this.uploadedDocuments.map((document) => document.documentId)
+        }
+      },
+    },
   },
   async updated() {
-    this.organizationReadOnly = JSON.parse(JSON.stringify(this.organization))
+    this.loadData()
   },
   methods: {
-    async update() {
-      this.$refs.organizationForm?.validate()
-      if (!this.isFormComplete) {
+    async loadData() {
+      try {
+        this.localLoading = true
+        this.organizationEdit = { ...this.organization, hasInclusionPolicy: this.organization.hasInclusionPolicy || false }
```

Comment:
I assume this is because the org is created with a null value instead of false?

Reply:
Correct.